test/versioned/elastic/dbtools.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2023 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+module.exports = function dbTools(client) {
```

Comment:
it looks like we're only using createIndex and createDocument now. do we want all of these?

Reply:
I'm fine ripping them out.